core/src/com/unciv/ui/objectdescriptions/BuildingDescriptions.kt

```diff
@@ -119,6 +119,11 @@ object BuildingDescriptions {
             if (value != originalBuilding[key])
                 yield(FormattedLine( key.name.tr() + " " +"[${value.toInt()}] vs [${originalBuilding[key].toInt()}]".tr(), indent=1))
 
+        val originalStatBonus = originalBuilding.getStatPercentageBonuses(null)
+        for ((key, value) in replacementBuilding.getStatPercentageBonuses(null))
+            if (value != originalStatBonus[key])
+                yield(FormattedLine("[${value.toInt()}]% ".tr() + key.name.tr() + " vs [${originalStatBonus[key].toInt()}]% ".tr() + key.name.tr(), indent = 1))
```

Comment:
> My guess - would need testing

Huh, that works. Tbh I was wondering if I'd need to add a new translation key, but I wasn't sure were to look for at first. Also, wow, so many of the languages don't translate "vs". Japanese is like, the only one that does

> if one names the common "key" once at the start of the line, the other twice after the numerics

Sorry, a bit lost here. Are you asking if it'll look better as "[stat] [value]% vs [original]%"?

Reply:
![Untitled](https://github.com/yairm210/Unciv/assets/127357473/779c4e79-0626-499a-94e3-1bb0b5cffd0f)
vs 
![Untitled2](https://github.com/yairm210/Unciv/assets/127357473/2aab9288-132a-4a0a-ba60-c1769236d75d)

For comparison, here's how it looks if there's also a stat change with a percent change:
![Untitled3](https://github.com/yairm210/Unciv/assets/127357473/3ea59466-e836-40a1-b2fc-ee250589c059)


Not sure if it looks a little too similar to the stats change variant or not. Otherwise, I guess this does look better. Putting the key at the end almost certainly might look off though, Imo

> translators have no idea what the entry means

Actually, even the ones that do translate it just converts it to a directional arrow. I don't think any language genuinely translates it currently. Though, I must say, the directional arrow looks nicer imo.

> did we say great writer's culture bomb

You're thinking of Great Musician's tourism blast, actually 🤓